components/WeatherCard.tsx

```diff
@@ -0,0 +1,109 @@
+import { FC } from 'react';
+import { useSiteData } from '~/lib/hooks';
+import useTime from '~/lib/hooks/useTime';
+
+import CloudyIcon from './icons/CloudyIcon';
+import SunnyIcon from './icons/SunnyIcon';
+import PartlyCloudyIcon from './icons/PartlyCloudyIcon';
```

Comment:
Instead of importing these separately, we should add these as exports into /icons/index.tsx! Then, we'll be able to import these all in one line.